tracer/build/_build/NukeExtensions/DotNetSettingsExtensions.cs

```diff
@@ -199,6 +199,18 @@ public static DotNetTestSettings WithMemoryDumpAfter(this DotNetTestSettings set
 
     public static DotNetTestSettings WithDatadogLogger(this DotNetTestSettings settings)
     {
+        try
+        {
+            if (string.Equals(Environment.GetEnvironmentVariable("DD_LOGGER_ENABLED"), "false", StringComparison.OrdinalIgnoreCase))
+            {
+                return settings;
+            }
+        }
+        catch
+        {
+            // Security issues...
+        }
```

Comment:
FWIW, we don't have to worry about the try-catch here (there's various other places we don't)